openedx/core/djangoapps/xblock/rest_api/views.py

```diff
@@ -147,6 +154,7 @@ def embed_block_view(request, usage_key: UsageKeyV2, view_name: str):
         'view_name': view_name,
         'is_development': settings.DEBUG,
         'oa_manifest': new_oa_manifest,
+        'title': title,
```

Comment:
Why is `show_title` optional? It seems weird to me if `"title": ""` is included in the response when you don't specify `show_title = True`. Could we just always include the title?

Reply:
And it might be clearer to call it `display_name` instead of `title` ?

Reply:
I think it looks good, updated in https://github.com/openedx/edx-platform/pull/37362/commits/9848257ecd051e7dd31fa743fccb10e21fb02f96. My concern is that this title is repeated in other components that are not texts. What do you think?

<img width="1131" height="649" alt="image" src="https://github.com/user-attachments/assets/33294e70-6ce0-47cd-81d6-dc568c7787d1" />

In the sidebar component, it would be repeated three times

<img width="526" height="860" alt="image" src="https://github.com/user-attachments/assets/0d15a6c6-6c52-4eb9-8bc9-c180e3b3af65" />

Reply:
Oh I see. I didn't really understand how this worked. I thought it was just returning data to the MFE as JSON, not rendering a template. 

In that case, I think `show_title` makes sense, but I'd prefer to just pass `show_title` as an additional variable into the template:

```
      {% if show_title  %}
        <div class="xblock-title">
            {{ display_name | safe }}
        </div>
      {% endif %}
```

But the way you have it now is fine too.

Reply:
Done: https://github.com/openedx/edx-platform/pull/37362/commits/fa2ee3c11329d95626b4547ffa1861aeb49ae2f3